app/assets/javascripts/components/common/ArticleViewer/containers/ArticleViewer.jsx

```diff
@@ -48,9 +50,15 @@ export class ArticleViewer extends React.Component {
     this.showBadArticleAlert = this.showBadArticleAlert.bind(this);
     this.submitBadWorkAlert = this.submitBadWorkAlert.bind(this);
     this.isWhocolorLang = this.isWhocolorLang.bind(this);
+    this.updateStateEditors = this.updateStateEditors.bind(this);
   }
 
   componentDidMount() {
+    if (this.props.assigned) {
+      if (!has(this.props.articleDetails, this.props.article.id) && isEmpty(this.props.fetchArticleDetails)) {
```

Comment:
This is confusing to me. What are the situations this code is trying to deal with? In particular, what does `isEmpty(this.props.fetchArticleDetails)` do?

Reply:
Hi, seems that there has been typographical error in the logic, i'll fix is immediately.